app/components/reviewComponent/review.tsx

```
@@ -0,0 +1,59 @@
+        </div>
+    );
+}
+
+
+{/* <Image
+src={gatheringImg}
+alt="모임 이미지"
+width={311}
+height={156}
+className="w-[311px] md:w-[280px] h-[156px] rounded-3xl"
+/> */}
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.

(lint/complexity/noUselessLoneBlockStatements)